homeassistant/components/android_ip_webcam/sensor.py

```diff
@@ -1,75 +1,177 @@
+        name="Audio Connections",
+        icon="mdi:speaker",
+        state_class=SensorStateClass.TOTAL,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda ipcam: ipcam.status_data.get("audio_connections"),
+    ),
+    AndroidIPWebcamSensorEntityDescription(
+        key="battery_level",
+        name="Battery Level",
+        device_class=SensorDeviceClass.BATTERY,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda ipcam: ipcam.export_sensor("battery_level")[0],
+        unit_fn=lambda ipcam: ipcam.export_sensor("battery_level")[1],
+    ),
+    AndroidIPWebcamSensorEntityDescription(
+        key="battery_temp",
+        name="Battery Temperature",
+        icon="mdi:thermometer",
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda ipcam: ipcam.export_sensor("battery_temp")[0],
+        unit_fn=lambda ipcam: ipcam.export_sensor("battery_temp")[1],
+    ),
+    AndroidIPWebcamSensorEntityDescription(
+        key="battery_voltage",
+        name="Battery Voltage",
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda ipcam: ipcam.export_sensor("battery_voltage")[0],
+        unit_fn=lambda ipcam: ipcam.export_sensor("battery_voltage")[1],
+    ),
+    AndroidIPWebcamSensorEntityDescription(
+        key="light",
+        name="Light Level",
+        icon="mdi:flashlight",
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda ipcam: ipcam.export_sensor("light")[0],
+        unit_fn=lambda ipcam: ipcam.export_sensor("light")[1],
+    ),
+    AndroidIPWebcamSensorEntityDescription(
+        key="motion",
+        name="Motion",
+        icon="mdi:run",
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda ipcam: ipcam.export_sensor("motion")[0],
+        unit_fn=lambda ipcam: ipcam.export_sensor("motion")[1],
+    ),
+    AndroidIPWebcamSensorEntityDescription(
+        key="pressure",
+        name="Pressure",
+        icon="mdi:gauge",
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda ipcam: ipcam.export_sensor("pressure")[0],
+        unit_fn=lambda ipcam: ipcam.export_sensor("pressure")[1],
+    ),
+    AndroidIPWebcamSensorEntityDescription(
+        key="proximity",
+        name="Proximity",
+        icon="mdi:map-marker-radius",
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda ipcam: ipcam.export_sensor("proximity")[0],
+        unit_fn=lambda ipcam: ipcam.export_sensor("proximity")[1],
+    ),
+    AndroidIPWebcamSensorEntityDescription(
+        key="sound",
```

Comment:
Light, motion, pressure, proximity and sound are not diagnostic sensors, right? They pick up features of the environment around the device, not diagnostic information about the device itself.